### PR TITLE
17.15 CEP Store Batch Feature

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds CEP-522, CEP-498, and CEP-518 to the 17.15 release notes. It also updates documentation.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
Reviewers, I hope you can take a look at this soon.

**Review by elof — APPROVED** [review]

File: docs/cep/table/index.md [hunk 1]
```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
 | attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
 | attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
-| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      | _from         | STRING              | Yes      |
-| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. | _to      | STRING              | Yes      |
+| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      |
+| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. |
```
Comment (StoyanTc) [inline docs/cep/table/index.md]: @diana-macrometa - The both parameters could be set only with `STORE` and should be move to the table below. 
Other store parameters are:
- `type`, e.g. type = 'database'
- `collection`, e.g. collection = "SampleCargoAppInputTable", 
- `collection.type`, e.g. collection.type="doc" , 
- `replication.type`, e.g. replication.type="global", 
- `map.type`, e.g. map.type='json'

File: docs/cep/table/index.md [hunk 2]
```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
...
 | [WITH (property_name = expression [, ...])] | Optional properties for the new table, such as a time-to-live or a partition key. |
 | SELECT select_expr [, ...] | The selection criteria for the new table. |
 | FROM from_stream … | The name of the existing stream to select data from. |
```
Comment (StoyanTc) [inline docs/cep/table/index.md]: @diana-macrometa - This is an explanation of the statements parts but not parameters.

File: docs/cep/table/index.md [hunk 3]
```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
 | attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
 | attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
```
Comment (StoyanTc) [inline docs/cep/table/index.md]: @diana-macrometa - This is an explanation what the attribute name/type is but not params.

File: docs/cep/table/index.md [hunk 4]
```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
```
Comment (StoyanTc) [inline docs/cep/table/index.md]: @diana-macrometa - This is a syntax sugar for `replication.type`.

**Review by StoyanTc — CHANGES_REQUESTED** [review]

File: docs/cep/table/index.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/cep/table/index.md]: I see. What should I call it, then? I think it is helpful to explain all the parts of our definitions.

File: docs/cep/table/index.md (hunk 4 above)
Comment (diana-macrometa) [reply docs/cep/table/index.md]: Updated.

File: docs/cep/table/index.md (hunk 2 above)
Comment (StoyanTc) [reply docs/cep/table/index.md]: @diana-macrometa - Please review my changes?

Comment (diana-macrometa) [reply docs/cep/table/index.md]: They look good to me. Is this PR ready to approve and merge?

**Review by StoyanTc — APPROVED** [review]

File: docs/cep/table/index.md (hunk 2 above)
Comment (StoyanTc) [reply docs/cep/table/index.md]: @diana-macrometa - Yes I think so since you're agree with my changes.

**Pull request merged**